# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-in-nested-shadow-dom.html
# Commit: 4f30d7afdf22
# Full Hash: 4f30d7afdf22c652481513564b60b37053c0948b
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-05-08 04:32:48
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Revert "Bug 1932150 - Fix build bustage" for causing build bustages.
#   
#   This reverts commit 0681b181d6a8a48ac460a97dacb7fca91f9989c3.
#   
#   Revert "Bug 1932150 - Fix lint failure" for causing Build bustage.
# ==============================================================================

diff -r 1e64a48e3517 -r 4f30d7afdf22 testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-in-nested-shadow-dom.html
--- a/testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-in-nested-shadow-dom.html	Wed May 07 20:11:46 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,27 +0,0 @@
-<!doctype html>
-<head>
-<link rel="match" href="cross-shadow-boundary-slot-in-nested-shadow-dom-ref.html"/>
-</head>
-<span id="outer">Outer</span>
-<div id="host">
-  <template shadowrootmode="open">
-    <span id="inner">Inner</span>
-    <div id="innerHost">
-      <template shadowrootmode="open">
-      <span id="inner">Inner2</span>
-      <slot></slot>
-      </template>
-      <span id="slotted">Slotted</span>
-    </div>
-  </template>
-</div>
-<script>
-  // This selects:
-
-  // er
-  // Inner
-  // Inner2 Slott
-  const slotted = host.shadowRoot.getElementById("slotted");
-  window.getSelection()
-      .setBaseAndExtent(outer.firstChild, 3, slotted.firstChild, 5);
-</script>